airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -261,6 +261,7 @@ class ConnectorLanguage(str, Enum):
     PYTHON = "python"
     JAVA = "java"
     LOW_CODE = "low-code"
+    MANIFEST_ONLY = "manifest-only"
```

Comment:
We've a lot of `if connector.language in [ConnectorLanguage.PYTHON, ConnectorLanguage.LOW_CODE]` in `airbyte-ci`, I'm afraid this change might silently filter out the `MANIFEST_ONLY` connectors from some `airbyte-ci` commands. 

The [`up-to-date` worfklow](https://github.com/airbytehq/airbyte/blob/master/.github/workflows/connectors_up_to_date.yml#L14) also uses a `--language` filter, can you please update it to run on this new connector language?

Reply:
Can certainly do.

@alafanechere Just to confirm the up-to-date command will bump the base image?

(since these manifest only connectors wont have a pyproject.toml to update)

Reply:
Yes it will bump the base image